packages/format-library/src/link/inline.js

```diff
@@ -180,8 +179,6 @@ function InlineLinkUI( {
 
 			newValue.start = newValue.end;
 
-			// Hides the Link UI.
-			newValue.activeFormats = [];
```

Comment:
Doing this means that when you _create_ the link and hit submit:

- the change gets submitted (we have to do this to persist the value).
- the UI remains open but switches to the editable state.
- the focus switches back to the Rich Text.

The later bit is the difficult part. I was thinking we might want to ensure focus remains in the open dialog. However, if we do that then each time a link is created users of assistive tech will need to 

- create link
- find a way to close the open dialog

Given that our dialogs don't have a means to do this _by default_, that poses a problem.

The alternative is what we have now which means users of assitive tech have focus transferred back to the rich text, but then they have this dialog floating around until such time as they leave the active format (the link "boundary" in the rich text).

I honestly don't know what is the best all around behaviour here.